### ISSUE TITLE
Latest API reference is for v1.11.0

### ISSUE THREAD

**Issue by eommc** [open-telemetry/opentelemetry-cpp#2994]
The latest documentation at https://opentelemetry-cpp.readthedocs.io is for version v1.11.0. 
Versions 1.12.0 and 1.13.0 exist, but they require a login https://readthedocs.org/projects/opentelemetry-cpp/versions/
Later versions don't exist at all, the build of the documentation has been failing for months. Looking at the build history, it's been failing since 25, 2023 up to January 2024, then there are no builds at all.
Is there any other place to see an API reference guide?

**Comment by lalitb** [comment]
Yes it's know issue, and was discussed in the community meeting. I was supposed to give access to other maintainers to debug further, somehow missed out. Let me have a look if it is easy to fix, else will ask for help from other maintainers.
